react/MuiCozyTheme/theme.js

```diff
@@ -91,17 +91,73 @@ export const theme = createMuiTheme({
       }
     },
     MuiListItem: {
+      root: {
+        borderTop: '1px solid var(--silver)',
+        minHeight: '4rem',
+        paddingTop: '.5rem',
+        paddingBottom: '.5rem',
+        '&:last-child': {
+          borderBottom: '1px solid var(--silver)'
+        },
+        '&$selected, &$selected:hover': {
+          backgroundColor: 'var(--zircon)'
+        },
+        '&:hover, &:focus': {
+          backgroundColor: 'var(--paleGrey)'
+        }
+      },
+      gutters: {
+        paddingLeft: '1rem',
+        paddingRight: '1rem'
```

Comment:
Should that change on mobile /

Reply:
Nope. Should be ok. This is the minimum size anyway. There could be higher paddings on desktop but it's really related to the app so I guess it's the app's job to decide its gutters.

Reply:
>  but it's really related to the app so I guess it's the app's job to decide its gutters.

IMHO it's not the app job to decide about it's list gutters on mobile/desktop. These decisions should be baked in the framework.

It's OK though to change it later when we need it.

Reply:
I meant it's not consistent throughout the different apps so I can't really find some common ground here. 
I'd rather get something clean that works by default and eventually let the app modify it if needed.